src/bookshelf/rebar.config

```diff
@@ -8,38 +8,45 @@
 
 {deps, [
         {lager,".*",
-         {git,"https://github.com/basho/lager", {tag, "2.1.1"}}},
+         {git,"https://github.com/basho/lager.git", {tag, "2.1.1"}}},
         {meck, ".*",
          {git, "https://github.com/eproxus/meck.git", {tag, "0.8.4"}}},
         {chef_secrets, ".*",
-         {git, "https://github.com/chef/chef_secrets", {branch, "master"}}},
+         %% ma/bump_versions is temporary until branch is merged.
+         {git, "https://github.com/chef/chef_secrets.git", {branch, "ma/bump_versions"}}},
         {iso8601, ".*",
-         {git, "https://github.com/erlsci/iso8601", {tag, "1.2.3"}}},
+         {git, "https://github.com/erlsci/iso8601.git", {tag, "1.2.3"}}},
         {opscoderl_wm, ".*",
          {git, "https://github.com/chef/opscoderl_wm.git", {branch,"master"}}},
         {erlsom, ".*",
-         {git, "https://github.com/chef/erlsom",
+         {git, "https://github.com/chef/erlsom.git",
           {branch, "integer_long_string_probs"}}},
```

Comment:
can we get on the master for this? looks like there is also a integer_long_string_probs2 branch that has some more updates.

Reply:
trying `integer_long_string_probs2`